spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/CounterController.java

```diff
@@ -53,7 +52,6 @@
 
 	public static final String COUNTER_PREFIX = "counter.";
 
-	@Autowired
```

Comment:
Thanks! - I see now... missed this one as a constructor arg. This is why field injection is such a bad idea, because otherwise this variable should be marked `final`. Will update.